private-api.go

```diff
@@ -39,6 +39,51 @@ type Balances struct {
 	Pair map[string]float64
 }
 
+func (c *Client) GetCompleteBalances(ctx context.Context) (CompleteBalances, error) {
+	resp, err := c.doPrivateAPIRequest(ctx, "returnCompleteBalances", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, errors.New(resp.Status)
+	}
+
+	var ret map[string]interface{}
```

Comment:
I wanna use Unmarshaler but Poloniex returns two pattern...

1. error

```
{"error":"<error message>"}
```

2. success

```
{"LTC":{"available":"5.015","onOrders":"1.0025","btcValue":"0.078"},"NXT:{...} ... }
```

How declare struct in this case?